src/modules/api/graphql-v2/resolvers/posts.ts

```diff
@@ -115,13 +115,25 @@ export class HivePost {
     console.log('possible_play_url', possible_play_url, images)
 
     const thumbnail_url = `https://media.3speak.tv/${this.permlink}/thumbnails/default.png`
+    let hasHeight = false
+    let hasWidth = false
+    let height = 0
+    let width = 0
+    if (json_metadata?.video?.info?.height !== null && json_metadata?.video?.info?.width !== null) {
+        hasHeight = true
+        hasWidth = true
+        height = json_metadata?.video?.info?.height
+        width = json_metadata?.video?.info?.width
+    }
     return {
       thumbnail_url: images.pop() || thumbnail_url,
       play_url: possible_play_url
         ? possible_play_url
         : `https://threespeakvideo.b-cdn.net/${this.rawDoc.permlink}/default.m3u8`,
       duration: json_metadata.video.info.duration,
-      is_short: json_metadata.video.info.duration < 60,
+      height: height,
+      width: width,
+      is_short: json_metadata.video.info.duration < 90 && hasHeight && hasWidth && height > width,
```

Comment:
I thought short is all videos under 90 seconds? Without the inclusion of width/height in the equation. Atleast that's how it was implemented in the legacy code. Not a big deal since I think in general we are shifting towards only vertical videos, but it might make sense to class it only as duration and filter separately by height.

Reply:
Understood, that wasn't in question above, just how exactly it's implemented. Small detail we will proceed as is.